netsuite/commonUtils.bal

```diff
@@ -110,7 +114,7 @@ isolated function getUpdateXMLBodyWithParentElement(string subElements) returns
     </soapenv:Envelope>`;
 }
 
-isolated function buildAddRecord(NewRecordType recordType, RecordCoreType recordCoreType, ConnectionConfig config) 
+isolated function getPayloadAddOp(NewRecordType recordType, RecordCoreType recordCoreType, ConnectionConfig config) 
```

Comment:
What is the meaning of `getPayloadAddOp`?

Shall we use a proper name for this?